src/components/ebay-section-title/examples/06-overflow/template.marko

```diff
@@ -0,0 +1,11 @@
+<ebay-section-title>
+    <ebay-section-title-title>Today’s Deals – All With Free Shipping</ebay-section-title-title>
+    <ebay-section-title-subtitle>Plus, guaranteed best prices.</ebay-section-title-subtitle>
+    <ebay-section-title-overflow>
```

Comment:
I would maybe make this section generic and call it like title-right-side or something.

Reply:
I'll check with Ian and Sean for naming suggestions as well. Keeping it as overflow for now, but I'm fine with renaming.

Reply:
I can @agliga's point, but I think overflow is okay. It's anything that "overflows" the section title. Also open to other suggestions though.

We definitely can't have direction specific names, like left or right, due to RTL. Things like "start" and "end" are better (taking a cue from CSS flexbox and Grid). As we build more of these layout kind of components I think we can re-review and look at consistency.